api-list.md

```diff
@@ -43,100 +61,115 @@ API Root Path
 
 `POST /users` - create a new user, needs to be coordinated with Cognito process
 
-`DELETE /users/:id` - delete a user, super admin level only? If we let users delete themselves, it would screw with existing and historical registrations.
+`DELETE /users/:id` - deactivate a user, super admin level only? If we let users delete themselves, it would screw with existing and historical registrations.
 `PUT /users/:id/deactivate` - alternative at user level?
 
 ## Organizations
 
 ---
 
-`GET organizations/` - return all orgs (super user and/or implement filtering)
-
-`GET organizations/:id` - return specific organization
+`GET /organizations` - return all orgs (super user and/or implement filtering)
 
-`POST organizations/` - create a new organization
+`GET /organizations/:id` - return specific organization
 
-### Nodes
+`POST /organizations` - create a new organization
 
-`GET organizations/:org_id/nodes` - return all nodes belonging to `:org_id`  
-`GET organizations/:org_id/nodes?isRoot=true` - return all root level nodes belonging to `:org_id`
+`DELETE /organizations/:id` - deactivate organization
 
-`GET organizations/:org_id/nodes/:id` - return node and children if present (same as /offerings/:id)
+### Nodes
 
-`PUT organizations/:org_id/nodes/:id` - update node
+`GET /organizations/:org_id/nodes` - return all nodes belonging to `:org_id`  
+`GET /organizations/:org_id/nodes?isRoot=true` - return all root level nodes belonging to `:org_id`
 
-`PUT organizations/:org_id/nodes/:id/publish` - set node as publicly visible  
-`PUT organizations/:org_id/nodes/:id/unpublish` - set node as invisible, will need lots of checking (can’t be done once offered, or part of an offered tree)
+`GET /organizations/:org_id/nodes/:id` - return node and children if present (same as /offerings/:id)
 
-`PUT organizations/:org_id/nodes/:id/offer` - set node to an offering (can be registered for)  
-`PUT organizations/:org_id/nodes/:id/deoffer` - not implemented? Not implemented once there are registrations?
+`PUT /organizations/:org_id/nodes/:id` - update node
```

Comment:
i think this is a more general way of updating nodes. any changes to a property are made by PUTing the value of that property on the resource rather than accessing a non-existent resource url